geode-core/src/integrationTest/java/org/apache/geode/internal/cache/PartitionedRegionIntegrationTest.java

```diff
@@ -49,10 +70,37 @@ public void bucketSorterShutdownAfterRegionDestroy() {
 
   @Test
   public void bucketSorterIsNotCreatedIfNoEviction() {
+    server.startServer();
     PartitionedRegion region =
         (PartitionedRegion) server.createRegion(RegionShortcut.PARTITION, "PR1",
             rf -> rf.setOffHeap(false));
     ScheduledExecutorService bucketSorter = region.getBucketSorter();
     assertThat(bucketSorter).isNull();
   }
+
+  @Test
+  // See GEODE-7106
```

Comment:
I don't think we follow adding ticket number in source/test code.

Reply:
Hey @agingade,
Thanks for reviewing this!.
Even though is not a "documented process", I saw a lot of places in the code and tests where we actually reference GEODE ticket numbers. I can remove it if you want, but I believe is a good practice to keep the context and history of changes.